common/src/main/java/org/corfudb/common/protocol/API.java

```diff
@@ -131,6 +137,71 @@ public static Response newPingResponse(Header header, ServerError error) {
                 .build();
     }
 
+    public static Request newRestartRequest(Header header) {
+        RestartRequest restartRequest = RestartRequest.getDefaultInstance();
+        return Request.newBuilder()
+                .setHeader(header)
+                .setRestartRequest(restartRequest)
+                .build();
+    }
+
+    public static Response newRestartResponse(Header header) {
+        return newRestartResponse(header, newNoServerError());
+    }
+
+    public static Response newRestartResponse(Header header, ServerError error) {
+        RestartResponse restartResponse = RestartResponse.getDefaultInstance();
+        return Response.newBuilder()
+                .setHeader(header)
+                .setError(error)
+                .setRestartResponse(restartResponse)
+                .build();
+    }
+
+    public static Request newResetRequest(Header header) {
+        ResetRequest resetRequest = ResetRequest.getDefaultInstance();
+        return Request.newBuilder()
+                .setHeader(header)
+                .setResetRequest(resetRequest)
+                .build();
+    }
+
+    public static Response newResetResponse(Header header) {
+        return newRestartResponse(header, newNoServerError());
+    }
```

Comment:
I think you forgot to change this to -
`return newResetResponse(header, newNoServerError());`

Reply:
Thanks for pointing this out